frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -777,6 +779,112 @@
 
         };
 
+        vm.getAllSubmissionResults = function(phaseId) {
+
+            vm.stopFetchingSubmissions = function() {
+                $interval.cancel(vm.poller);
+            };
+
+            vm.stopFetchingSubmissions();
+            vm.isResult = true;
+            vm.phaseId = phaseId;
+
+            // loader for loading submissions.
+            vm.startLoader =  loaderService.startLoader;
+            vm.startLoader("Loading Submissions");
+
+            // get submissions of all the challenge phases
+            vm.isNext = '';
+            vm.isPrev = '';
+            vm.currentPage = '';
+            vm.showPagination = false;
+
+            var parameters = {};
+            parameters.url = "challenges/" + vm.challengeId + "/challenge_phase/" + vm.phaseId + "/submissions/";
+            parameters.method = 'GET';
+            parameters.data = {};
+            parameters.token = userKey;
+            parameters.callback = {
+                onSuccess: function(response) {
+                    var details = response.data;
+
+                    vm.submissionResult = details;
+
+                    if (vm.submissionResult.count === 0) {
```

Comment:
I am not able to understand the logics of these `if else` here. Can we replace them smartly by ternary operators?